vita3k/config/include/config/config.h

```diff
@@ -151,6 +151,8 @@ enum PerfomanceOverleyPosition {
     code(bool, "spirv-shader", false, spirv_shader)                                                     \
     code(bool, "fps-hack", false, fps_hack)                                                             \
     code(uint64_t, "current-ime-lang", 4, current_ime_lang)                                             \
+    code(bool, "acceleration-and-gyroscope", true, tiltsens)                                            \
```

Comment:
For name of option using same for both, with jyste replace - by _ in left, like other